packages/design-system-scripts/gulp/docs/generatePages/generateHtmlExample.js

```diff
@@ -33,8 +33,8 @@ function processMarkup(markup, modifier) {
  * @param {String} rootPath - Root docs site path
  * @return {Promise}
  */
-function generateHtmlExample(page, modifier, docsPath, { rootPath }) {
-  if (rootPath) rootPath = `${rootPath}/`;
+function generateHtmlExample(page, modifier, docsPath, options) {
+  const rootPath = (options.rootPath) ? `${options.rootPath}/` : null;
```

Comment:
Same thing here as in generateDocsPage: It doesn't look like `rootPath` being null is used anywhere else in this function but to allow it to fall through to `|| ''`; could we just use an empty string here instead of null and get rid of the `|| ''` below?